src/gpodder/gtkui/main.py

```diff
@@ -2921,33 +2921,34 @@ def on_goto_mygpo(self, widget):
         self.mygpo_client.open_website()
 
     def on_download_subscriptions_from_mygpo(self, action=None):
+        def after_login():
+            dir = gPodderPodcastDirectory(self.gPodder,
+                                           _config=self.config,
+                                           custom_title=_('Subscriptions on gpodder.net'),
```

Comment:
Could we here substitute the actual url used instead of always showing "gpodder.net"?